docs/guides/flavors/ios.md

```diff
@@ -0,0 +1,190 @@
+---
+sidebar_position: 2
+title: iOS | Flavors
+sidebar_label: 🍎 iOS
+description: Push patches to multiple deployment tracks on iOS
+---
+
+# Flavors
+
+This guide will walk through how to setup an app in which there are 2 deployment tracks: `internal` and `stable`. It will cover how to validate a patch on the internal track and then promote the patch to the stable track on iOS.
+
+## Prerequisites
+
+This guide assumes the Shorebird CLI is installed on your machine and that you are logged into an account. Refer to the [getting started](/) instructions for more information.
```

Comment:
`shorebird` command? instead of "Shorebird CLI"?  I question how many people know the CLI TLA.